subsys/net/lib/sockets/Kconfig

```diff
@@ -0,0 +1,28 @@
+# Kconfig - BSD Sockets like API
+
+#
+# Copyright (c) 2017 Linaro Limited.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig NET_SOCKETS
+	bool "BSD Sockets like API"
+	default n
+	help
+	Provide BSD Sockets like API on top of native Zephyr networking API.
+
+if NET_SOCKETS
+
+config NET_SOCKETS_POSIX_NAMES
+        bool "Standard POSIX names for Sockets API"
+        default n
+        help
+        By default, Sockets API function are prefixed with "zsock_" to avoid
+        namespacing issues. If this option is enabled, they will be provided
+        with standard POSIX names like socket(), recv(), and close(), to help
+        with porting existing code. Note that close() may require a special
+        attention, as in POSIX it closes any file descriptor, while with this
+        option enaled, it will still apply only to sockets.
```

Comment:
Sure, Zephyr is being used together with a 3rd-party library, whose authors also doubted that namespace clashes would occur, so used names like "listen" and "close" in their API ;-).